src/payment-flows/nonce.js

```diff
@@ -65,17 +65,14 @@ function initNonce({ props }) : PaymentFlowInstance {
     const { createOrder, clientID, wallet, branded } = props;
     let { paymentMethodNonce } = props;
 
-    // $FlowFixMe
-    getLogger().info(paymentMethodNonce);
-
     // eslint-disable-next-line no-warning-comments
     // TODO: remove check when reading from wallet
     if (!paymentMethodNonce)  {
         paymentMethodNonce = wallet.card.instruments[0].tokenID;
     }
 
     const start = () => {
-        getLogger().info('start payment with nonce');
```

Comment:
Usually these event names are named_like_this because they're CAL event names. can you re-name it to `start_payment_with_nonce`?